util/latlng_recon_service.py

```diff
@@ -16,13 +16,48 @@
 See latlng_recon_service_test.py for usage example.
 """
 
+import concurrent.futures
+import logging
 import requests
+from typing import Callable, Dict, List, NewType, TypeVar, Tuple
 
-_RECON_ROOT = 'https://api.datacommons.org/v1/recon/resolve/coordinate'
+LatLng = NewType('LatLng', Tuple[float, float])
```

Comment:
Since there isn't much guidance on this, I wonder if we should use the suffix Type for these to distinguish from vars / classes (e.g. LatLngType, etc). Wdyt?

Nit: DCID --> DcidType.

Reply:
We should also make these module private for now (add an underscore prefix).

Reply:
>  I wonder if we should use the suffix Type 

Good idea, added Type suffix. Since some of these types are used for function arguments, which are called externally, they need to be exposed as opposed to being private?

Reply:
Ah got it. Ok, then it's better if exposed with the Type suffix.